pyfixest/estimation/feols_.py

```diff
@@ -1503,8 +1503,14 @@ def tidy(self, alpha=0.05) -> pd.DataFrame:
             A tidy pd.DataFrame containing the regression results, including point
             estimates, standard errors, t-statistics, and p-values.
         """
-        _coefnames = self._coefnames
+        if alpha is None:
+            ub, lb = 0.975, 0.025
+            self.get_inference()
```

Comment:
I recall the get_inference as if the user had inputted non-default alpha for tidy previously, then the user recall tidy again, the value of confidence interval will be the non-default eventho the dataframe columns will show a default confidence interval

What do you think? @s3alfisc